Src/WindowsServer/WindowsServer.Nuget.Tests/TelemetryInitailizersTests.cs

```diff
@@ -24,17 +24,16 @@ public void InstallAddsAzureRoleEnvironmentTelemetryInitializer()
         }
 
         [TestMethod]
-        public void InstallAddsDomainNameRoleInstanceTelemetryInitializer()
+        public void InstallAddsAzureWebAppTelemetryInitializer()
         {
             string emptyConfig = ConfigurationHelpers.GetEmptyConfig();
             XDocument configAfterTransform = ConfigurationHelpers.InstallTransform(emptyConfig);
 
-            Type typeToFind = typeof(DomainNameRoleInstanceTelemetryInitializer);
+            Type typeToFind = typeof(AzureWebAppRoleEnvironmentTelemetryInitializer);
+
+            var nodes = ConfigurationHelpers.GetTelemetryInitializers(configAfterTransform).Descendants().ToList();
```

Comment:
Two comments:
1) Why add the convert to list?  You don't need/use a list that I can see.
2) Using "var" for the return of a function call isn't very readable